### ISSUE TITLE
Codelyzer Accessibility Rule Violations Not Showing in VSCode Editor Window

### ISSUE THREAD

**Issue by christopherallanperry** [mgechev/codelyzer#956]
**Describe the bug**

In order to include accessibility checks in an Angular 9.0 project, I followed the configuration steps in this article - https://web.dev/accessible-angular-with-codelyzer/

Subsequent tests show that the rules are applied using `ng lint` from the CLI, but are not highlighted as a warning, nor is the descriptor popup  available.

TSLint warnings are shown as normal, but not the accessibility ones.

**Context and configuration**

### ,project_root>/tslint.json
```
{
  "rulesDirectory": ["/node_modules/codelyzer"],
  "rules": {
    "arrow-return-shorthand": true,
    "callable-types": true,
    "class-name": true,
    "comment-format": [true, "check-space"],
    "curly": true,
    "deprecation": {
      "severity": "warn"
    },
    "eofline": true,
    "forin": true,
    "import-blacklist": [true, "rxjs/Rx"],
    "import-spacing": true,
    "indent": [true, "spaces"],
    "interface-over-type-literal": true,
    "label-position": true,
    "max-line-length": [true, 140],
    "member-access": false,
    "member-ordering": [
      true,
      {
        "order": [
          "static-field",
          "instance-field",
          "static-method",
          "instance-method"
        ]
      }
    ],
    "no-arg": true,
    "no-bitwise": true,
    "no-console": [true, "debug", "info", "time", "timeEnd", "trace"],
    "no-construct": true,
    "no-debugger": true,
    "no-duplicate-super": true,
    "no-empty": false,
    "no-empty-interface": true,
    "no-eval": true,
    "no-inferrable-types": [true, "ignore-params"],
    "no-misused-new": true,
    "no-non-null-assertion": true,
    "no-shadowed-variable": true,
    "no-string-literal": false,
    "no-string-throw": true,
    "no-switch-case-fall-through": true,
    "no-trailing-whitespace": true,
    "no-unnecessary-initializer": true,
    "no-unused-expression": true,
    "no-use-before-declare": true,
    "no-var-keyword": true,
    "object-literal-sort-keys": false,
    "one-line": [
      true,
      "check-open-brace",
      "check-catch",
      "check-else",
      "check-whitespace"
    ],
    "prefer-const": true,
    "quotemark": [true, "single"],
    "radix": true,
    "semicolon": [true, "always"],
    "triple-equals": [true, "allow-null-check"],
    "typedef-whitespace": [
      true,
      {
        "call-signature": "nospace",
        "index-signature": "nospace",
        "parameter": "nospace",
        "property-declaration": "nospace",
        "variable-declaration": "nospace"
      }
    ],
    "unified-signatures": true,
    "variable-name": false,
    "whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-operator",
      "check-separator",
      "check-type"
    ],
    "no-output-on-prefix": true,
    "use-input-property-decorator": true,
    "use-output-property-decorator": true,
    "use-host-property-decorator": true,
    "no-input-rename": true,
    "no-output-rename": true,
    "use-life-cycle-interface": true,
    "use-pipe-transform-interface": true,
    "component-class-suffix": true,
    "directive-class-suffix": true,
    "template-accessibility-alt-text": true,
    "template-accessibility-elements-content": true,
    "template-accessibility-label-for": true,
    "template-accessibility-tabindex-no-positive": true,
    "template-accessibility-table-scope": true,
    "template-accessibility-valid-aria": true,
    "template-click-events-have-key-events": true,
    "template-mouse-events-have-key-events": true,
    "template-no-autofocus": true,
    "template-no-distracting-elements": true
  }
}
```

### <project_root>/src/tslint.json
```
{
  "extends": "../tslint.json",
  "rulesDirectory": ["../node_modules/codelyzer"],
  "rules": {
    "directive-selector": [true, "attribute", "app", "camelCase"],
    "component-selector": [true, "element", "app", "kebab-case"],
    "template-accessibility-alt-text": true,
    "template-accessibility-elements-content": true,
    "template-accessibility-label-for": true,
    "template-accessibility-tabindex-no-positive": true,
    "template-accessibility-table-scope": true,
    "template-accessibility-valid-aria": true,
    "template-click-events-have-key-events": true,
    "template-mouse-events-have-key-events": true,
    "template-no-autofocus": true,
    "template-no-distracting-elements": true
  }
}
```

### <project_root>/tsconfig.json
```
{
  "compileOnSave": false,
  "compilerOptions": {
    "baseUrl": "./",
    "outDir": "./dist/out-tsc",
    "sourceMap": true,
    "declaration": false,
    "moduleResolution": "node",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "target": "es5",
    "typeRoots": [
      "node_modules/@types"
    ],
    "lib": [
      "es2017",
      "dom"
    ]
  }
}
```

### user/settings.json
```
{
    "workbench.startupEditor": "newUntitledFile",
    "atomKeymap.promptV3Features": true,
    "editor.multiCursorModifier": "ctrlCmd",
    "editor.formatOnPaste": true,
    "window.zoomLevel": 0,
    "editor.formatOnSave": true,
    "editor.formatOnType": true,
    "peacock.favoriteColors": [
        {
            "name": "Angular Red",
            "value": "#b52e31"
        },
        {
            "name": "Auth0 Orange",
            "value": "#eb5424"
        },
        {
            "name": "Azure Blue",
            "value": "#007fff"
        },
        {
            "name": "C# Purple",
            "value": "#68217A"
        },
        {
            "name": "Gatsby Purple",
            "value": "#639"
        },
        {
            "name": "Go Cyan",
            "value": "#5dc9e2"
        },
        {
            "name": "Java Blue-Gray",
            "value": "#557c9b"
        },
        {
            "name": "JavaScript Yellow",
            "value": "#f9e64f"
        },
        {
            "name": "Mandalorian Blue",
            "value": "#1857a4"
        },
        {
            "name": "Node Green",
            "value": "#215732"
        },
        {
            "name": "React Blue",
            "value": "#00b3e6"
        },
        {
            "name": "Something Different",
            "value": "#832561"
        },
        {
            "name": "Vue Green",
            "value": "#42b883"
        }
    ],
    "[javascript]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[typescript]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[html]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[css]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "[json]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
    "explorer.confirmDragAndDrop": false,
    "javascript.preferences.quoteStyle": "double",
    "typescript.preferences.quoteStyle": "double",
    "explorer.confirmDelete": false,
    "git.autofetch": true,
    "workbench.iconTheme": "material-icon-theme",
    "tslint.rulesDirectory": [
        "/node_modules/codelyzer"
    ],
    "typescript.tsdk": "node_modules/typescript/lib"
}
```

### <project_root>/.vscode/settings.json (workspace)
```
{}
```

**To Reproduce**

1. Create default Angular project using the Angular CLI
2. Configure JSON files described above
3. Insert `<label class="label">Label</label>` into the default `app.component.html` file
4. From the CLI, run `ng lint` // Accessibility warnings are reported
5. In the VSCode editor, no warnings are shown

**Expected behavior**

Codelyzer accessibility warnings are shown via TSLint

**Code**

See above

**Environment**
The project is running on a current Windows 10 build with the following installed:
- VSCode v1.42.1
- Node v12.16
- Angular ^9.0.0
- Codelyzer ^5.2.1
- TSLint ~6.0.0

**Additional context**

One thing I have noticed is that VSCode doesn't appear to recognise the `"tslint.rulesDirectory": ["/node_modules/codelyzer"]` as being valid, even though it was located using Path Intellisense. As you can see from the attached image, it is 'greyed' out compared to the key/value pairs surrounding it.

![tslint](https://user-images.githubusercontent.com/10407622/74852754-f14d4800-5334-11ea-9d78-f38ec15430e2.PNG)

**Comment by mgechev** [comment]
There was a discussion with the maintainers of the tslint extension for VSCode on how to implement this. Unfortunately, there's nothing on codelyzer's side we can do.

**Comment by christopherallanperry** [comment]
Hi Minko,

Unfortunate, but thanks for letting me know.

Chris
